tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.js

```diff
@@ -475,17 +514,19 @@ const clickTab = async ( context, tabPosition ) => {
 	await context.locator( `.elementor-widget-n-tabs .e-n-tab-title >> nth=${ tabPosition } ` ).first().click();
 };
 
-async function setup( wpAdmin ) {
+async function setup( wpAdmin, svgActive = false ) {
```

Comment:
I don't think this is a good implementation, this to specific and covers only one use case.

Can you try this instead?
The suggestion bellow is more generic.
This way we can add any experiment we want and we are not limited to SVG icons expiriment. 

```
async function setup( wpAdmin, customExperiment = '' ) {
    let experiments = {
        container: 'active',
        'nested-elements': 'active',
    };

    experiments = { ...experiments, ...customExperiment };
    await wpAdmin.setExperiments( experiments );
}

async function cleanup( wpAdmin, customExperiment = '' ) {
    let experiments = {
        container: 'inactive',
        'nested-elements': 'inactive',
    };

    experiments = { ...experiments, ...customExperiment };
    await wpAdmin.setExperiments( experiments );
}
```

The implementation should look like:


```
        await setup( wpAdmin, { e_font_icon_svg: 'active' } );
        await cleanup( wpAdmin, { e_font_icon_svg: 'inactive' } );
```

Reply:
Implemented